[2022-08-06 19:36:26,241] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [queued]>
[2022-08-06 19:36:26,259] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [queued]>
[2022-08-06 19:36:26,259] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:36:26,259] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:36:26,259] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:36:26,280] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-09-08 00:00:00+00:00
[2022-08-06 19:36:26,287] {standard_task_runner.py:52} INFO - Started process 498 to run task
[2022-08-06 19:36:26,290] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-09-08T00:00:00+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpdtm2_up8', '--error-file', '/tmp/tmpubtqgnoo']
[2022-08-06 19:36:26,290] {standard_task_runner.py:80} INFO - Job 130: Subtask second_task
[2022-08-06 19:36:26,369] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:36:26,505] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-08T00:00:00+00:00
[2022-08-06 19:36:26,506] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:36:26,507] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:36:26,534] {subprocess.py:85} INFO - Output:
[2022-08-06 19:36:26,536] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:36:26,537] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:36:26,615] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210908T000000, start_date=20220806T193626, end_date=20220806T193626
[2022-08-06 19:36:26,662] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:36:26,727] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:50:17,250] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [queued]>
[2022-08-06 21:50:17,268] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [queued]>
[2022-08-06 21:50:17,268] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:17,268] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:50:17,268] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:17,288] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-09-08 00:00:00+00:00
[2022-08-06 21:50:17,295] {standard_task_runner.py:52} INFO - Started process 672 to run task
[2022-08-06 21:50:17,298] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-09-08T00:00:00+00:00', '--job-id', '127', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp0nhpswkd', '--error-file', '/tmp/tmpisx_3kib']
[2022-08-06 21:50:17,298] {standard_task_runner.py:80} INFO - Job 127: Subtask second_task
[2022-08-06 21:50:17,372] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:50:17,455] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-08T00:00:00+00:00
[2022-08-06 21:50:17,455] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:50:17,456] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:50:17,468] {subprocess.py:85} INFO - Output:
[2022-08-06 21:50:17,470] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:50:17,470] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:50:17,499] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210908T000000, start_date=20220806T215017, end_date=20220806T215017
[2022-08-06 21:50:17,551] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:50:17,581] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:55:47,623] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [queued]>
[2022-08-06 21:55:47,638] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [queued]>
[2022-08-06 21:55:47,638] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:55:47,638] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:55:47,638] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:55:47,658] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-09-08 00:00:00+00:00
[2022-08-06 21:55:47,664] {standard_task_runner.py:52} INFO - Started process 471 to run task
[2022-08-06 21:55:47,668] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-09-08T00:00:00+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp2zv73b_v', '--error-file', '/tmp/tmp0umkout_']
[2022-08-06 21:55:47,668] {standard_task_runner.py:80} INFO - Job 126: Subtask second_task
[2022-08-06 21:55:47,728] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:55:47,815] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-08T00:00:00+00:00
[2022-08-06 21:55:47,815] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:55:47,816] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:55:47,828] {subprocess.py:85} INFO - Output:
[2022-08-06 21:55:47,830] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:55:47,830] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:55:47,858] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210908T000000, start_date=20220806T215547, end_date=20220806T215547
[2022-08-06 21:55:47,879] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:55:47,909] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:10:49,863] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [queued]>
[2022-08-06 22:10:49,881] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [queued]>
[2022-08-06 22:10:49,881] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:10:49,881] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:10:49,881] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:10:49,902] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-09-08 00:00:00+00:00
[2022-08-06 22:10:49,909] {standard_task_runner.py:52} INFO - Started process 488 to run task
[2022-08-06 22:10:49,916] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-09-08T00:00:00+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpsb1qsiyd', '--error-file', '/tmp/tmpc1cf6n8y']
[2022-08-06 22:10:49,917] {standard_task_runner.py:80} INFO - Job 130: Subtask second_task
[2022-08-06 22:10:50,018] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:10:50,128] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-08T00:00:00+00:00
[2022-08-06 22:10:50,129] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:10:50,130] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:10:50,145] {subprocess.py:85} INFO - Output:
[2022-08-06 22:10:50,146] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:10:50,147] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:10:50,197] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210908T000000, start_date=20220806T221049, end_date=20220806T221050
[2022-08-06 22:10:50,246] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:10:50,299] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:18:36,638] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [queued]>
[2022-08-06 22:18:36,654] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [queued]>
[2022-08-06 22:18:36,655] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:18:36,655] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:18:36,655] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:18:36,676] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-09-08 00:00:00+00:00
[2022-08-06 22:18:36,683] {standard_task_runner.py:52} INFO - Started process 496 to run task
[2022-08-06 22:18:36,686] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-09-08T00:00:00+00:00', '--job-id', '127', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpys5yd9x0', '--error-file', '/tmp/tmp_hfxusy8']
[2022-08-06 22:18:36,686] {standard_task_runner.py:80} INFO - Job 127: Subtask second_task
[2022-08-06 22:18:36,757] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [running]> on host ed53bde1c44a
[2022-08-06 22:18:36,877] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-08T00:00:00+00:00
[2022-08-06 22:18:36,878] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:18:36,879] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:18:36,893] {subprocess.py:85} INFO - Output:
[2022-08-06 22:18:36,895] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:18:36,895] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:18:36,983] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210908T000000, start_date=20220806T221836, end_date=20220806T221836
[2022-08-06 22:18:37,018] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:18:37,061] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:22:11,414] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [queued]>
[2022-08-06 22:22:11,428] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [queued]>
[2022-08-06 22:22:11,428] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:22:11,428] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:22:11,428] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:22:11,445] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-09-08 00:00:00+00:00
[2022-08-06 22:22:11,451] {standard_task_runner.py:52} INFO - Started process 527 to run task
[2022-08-06 22:22:11,454] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-09-08T00:00:00+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpba8q93gd', '--error-file', '/tmp/tmp_2a0kp6f']
[2022-08-06 22:22:11,454] {standard_task_runner.py:80} INFO - Job 126: Subtask second_task
[2022-08-06 22:22:11,532] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-09-08T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:22:11,646] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-08T00:00:00+00:00
[2022-08-06 22:22:11,650] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:22:11,651] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:22:11,669] {subprocess.py:85} INFO - Output:
[2022-08-06 22:22:11,671] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:22:11,671] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:22:11,788] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210908T000000, start_date=20220806T222211, end_date=20220806T222211
[2022-08-06 22:22:11,826] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:22:11,872] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
